### ISSUE TITLE
gnt-instance add does not warn about non-existing networks

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#666]
Originally reported of Google Code with ID 610.

```
<b>What steps will reproduce the problem?</b>
1. create a network with gnt-network
2. connect the network to the default node group
3. create an instance in that node group, specifying the nic parameter with a nonexisting network (e.g. --net 0:network=idontexist)

<b>What is the expected output? What do you see instead?</b>

Ganeti should warn the user that he is trying to connect the instance to a non-existing network. Instead, the instance is created and the default nic parameters are used.



:~# gnt-network list
Network Subnet         Gateway     MacPrefix GroupList              Tags
mynet2  192.168.2.0/24 192.168.2.1 -         default (bridged, br0)


:~# gnt-instance add -t drbd -s 1G -o debian-image --net 0:network=mynet1 INSTANCE
Job 17 is trying to acquire all necessary locks
Thu Nov 14 09:32:47 2013  - INFO: Selected nodes for instance INSTANCE via iallocator hail: NODE1, NODE2
Thu Nov 14 09:32:48 2013 * creating instance disks...
Thu Nov 14 09:32:52 2013 adding instance INSTANCE to cluster config
Thu Nov 14 09:32:52 2013  - INFO: Waiting for instance INSTANCE to sync disks
Thu Nov 14 09:32:52 2013  - INFO: - device disk/0:  2.40% done, 57s remaining (estimated)
Thu Nov 14 09:33:49 2013  - INFO: Instance INSTANCE's disks are in sync
Thu Nov 14 09:33:49 2013 * running the instance OS create scripts...
Thu Nov 14 09:33:52 2013 * starting instance...

:~# gnt-instance info INSTANCE 
- Instance name: INSTANCE
  [...]
  NICs: 
    - nic/0: 
      MAC: aa:00:00:9e:49:31
      IP: None
      mode: bridged
      link: brDEFAULT
      vlan: {'Nothing': None}
      network: None
      UUID: bbcd359b-7d53-4565-808b-92ce87dad715
      name: None


```
Originally added on 2013-11-14 09:42:29 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Moving all issues that won't be addressed in 2.11 anymore to 2.12.
```
Originally added on 2014-05-14 11:28:37 +0000 UTC.

Added to Milestone: Release2.12
Removed from Milestone: Release2.11

**Comment by Ganeti-Issues-Migrator** [comment]
```
I am closing this issue because it's old and I was not able to replicate this with the latest version of Ganeti. This is according to our current team effort to migrate to Github issues. Please feel free to comment and re-open this if you think it's still relevant.

See https://groups.google.com/forum/#!topic/ganeti/b2-eukpsXpk for more details.

```
Originally added on 2016-12-15 11:33:24 +0000 UTC.

Changed State: Obsolete
Removed from Milestone: Release2.12
